### PR TITLE
Dropdown update

### PR DESCRIPTION
**Pull request by vergilius**
Update Dropdown behaviour and expose toggling and isOpened property.
Doesn't have impact on Dropdowns used in style-guide.

You will use it like this:

```
<OpenedController>
      {
            ({isOpened, toggle}) =>
                  <Dropdown
                      {...defaultProps}
                      isOpened={isOpened}
                      toggle={toggle}
                    />
      }
</OpenedController>
```

See referenced PR's for details

### CODE REVIEW

File: src/components/opened-controller/OpenedController.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React, {Component, Fragment} from 'react';
+import PropTypes from 'prop-types';
+
+export class OpenedController extends Component {
+  state = {
+    isOpened: false
+  };
+
```
Comment (vergilius) [inline src/components/opened-controller/OpenedController.jsx]: componentWIllMount && unmount - add/remove body click listeners

File: src/components/opened-controller/OpenedController.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    if (this.props.onOpen && !this.state.isOpened) {
+      this.props.onOpen();
+    }
+
+    this.setState(prevState => ({isOpened: !prevState.isOpened}));
+  };
+
+  render() {
+    return (
+      <Fragment>
```
Comment (aju) [inline src/components/opened-controller/OpenedController.jsx]: you even don't need fragment I think! :)

**Comment by kdzwinel** [comment]
LGTM but `<OpenedController>` should be moved to CWB

**Comment by vergilius** [comment]
Should be moved to CWB

**Pull request merged**